allenact_plugins/ithor_plugin/ithor_environment.py

```diff
@@ -639,7 +639,7 @@ def _snap_agent_to_initially_reachable(self, verbose=True):
             )
 
     def step(
-        self, action_dict: Dict[str, Union[str, int, float]]
+        self, **action_dict: Dict[str, Union[str, int, float]]
```

Comment:
shouldn't the new type be just `Union[str, int, float]`?

Reply:
I don't think so. If one passes in
```python
self.env.step(action="MoveAhead", moveMagnitude=0.25)
```
`action_dict` now gets interpreted as:
```python
action_dict = dict(action="MoveAhead", moveMagnitude=0.25)
```
But one shouldn't be able to just pass in
```python
self.env.step("MoveAhead", 0.25)
```
or something like this, without the keywords.

Reply:
Maybe I'm misinterpreting it, but shouldn't the type for the kwargs not be a Dict?
[pep-0484](https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values)

Reply:
Sorry, missed this. I think you're right, `**kwargs` should be annotated with the types and not as a dict. Fixed in https://github.com/allenai/allenact/pull/290/commits/33a8f40da7aa924c3e3682c1edd3b002f421ec02.